app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, jsonify, request
+
+app = Flask(__name__)
+
+events = [
+    {
+        "dates": [
+            {
+                "day": 4,
+                "month": 2,
+                "times": {
+                    "1": {
+                        "end": 12,
+                        "start": 10
+                    },
+                    "2": {
+                        "end": 23,
+                        "start": 22
+                    }
+                },
+                "year": 2020
+            }
+        ],
+        "description": "first description",
+        "title": "event 1"
+    }
+]
+
+
+@app.route("/events/new", methods=["POST"])
```

Comment:
`/event(s)/` could be a better option.
More info:
* https://restfulapi.net/resource-naming/
* https://www.restapitutorial.com/lessons/restfulresourcenaming.html